### ISSUE TITLE
Great plugin. A few suggestions

### ISSUE THREAD

**Issue by NML32** [SavionLegendZzz/UUIDBanning#3]
This is a idea for banning really bad people. 
A few things I noticed and suggestions.
Q: How do you Pardon?
Q: Could you enable the command through the console?

In the config file the data is encrypted so it makes it hard to remove entries or to verify if someone has been UUID banned.

Thanks for taking the time to review my comments.

**Comment by SavionLegendZzz** [comment]
i tried to make it where it looked like Steve:{hhis/her uuid} but it didnt work but i will try and find out when i am done working on my UHC plugin

**Comment by NML32** [comment]
Thank you that would be great.

**Comment by linuzo** [comment]
Yeah, we definately need to be able to pardon someone..  But great plugin.
